### ISSUE TITLE
[New/Revise] Show dates in accuracy graph

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#737]
## Summary of this new feature
Currently, x-axis is time steps (0, 1, 2, 3,...) of the selected phase in the graph of estimation accuracy. It is prefered to show date as x-axis. This is related to #503.

```Python
import covsirphy as cs
data_loader = cs.DataLoader()
jhu_data = data_loader.jhu()
population_data = data_loader.population()
snl = cs.Scenario(country="Japan", province=None)
snl.register(jhu_data, population_data)
snl.trend(show_figure=False)
snl.estimate(cs.SIRF, phases=["10th"])
snl.estimate_accuracy(phase="10th")
```
CovsirPhy version: 2.19.1-gamma-fu3

![Figure_2](https://user-images.githubusercontent.com/7270139/115990448-41413200-a5fe-11eb-9b43-d5020d56a224.png)

**Comment by lisphilar** [comment]
- "predicted" is not the correct word for simulation, "simulated" will be used.
- Model name is not necessary for accuracy plot because model is fixed for all phases.

**Comment by lisphilar** [comment]
This was closed with #743 and #751 (version 2.19.1-zeta-fu2).
https://gist.github.com/lisphilar/fa040a9f2f066b44a6441f5f13c89171
